internal/namespaces/instance/v1/instance_cli.go

```diff
@@ -438,25 +439,31 @@ func instanceServerUpdate() *core.Command {
 				EnumValues: []string{},
 			},
 			{
-				Name:       "volumes.{key}.id",
+				Name:       "volumes.{key}.key",
+				Short:      ``,
+				Required:   false,
+				EnumValues: []string{},
+			},
+			{
+				Name:       "volumes.{key}.value.id",
```

Comment:
same as https://github.com/scaleway/scaleway-cli/pull/591, generation should be fixed before